fe/fe-core/src/main/java/com/starrocks/qe/scheduler/dag/FragmentInstance.java

```diff
@@ -309,10 +309,9 @@ public int getTableSinkDop() {
         } else {
             int sessionVarSinkDop = ConnectContext.get().getSessionVariable().getPipelineSinkDop();
             if (sessionVarSinkDop > 0) {
-                return Math.min(dop, sessionVarSinkDop);
-            } else {
-                return Math.min(dop, IcebergTableSink.ICEBERG_SINK_MAX_DOP);
+                return sessionVarSinkDop;
             }
+            return dop;
```

Comment:
do you need to preserve old behaviour  ?
`min(dop, ICEBERG_SINK_MAX_DOP)`

Reply:
I will give it a cap after more rigorous testing. But now this is ok.